cpmpy/transformations/linearize.py

```diff
@@ -445,6 +435,66 @@ def only_positive_bv(lst_of_expr):
 
     return newlist
 
+def only_positive_bv_sub(expr):
+    """
+        Replaces a linear expression containing :class:`~cpmpy.expressions.variables.NegBoolView` with an equivalent expression 
+        using only :class:`~cpmpy.expressions.variables.BoolVar`. 
+        
+        Arguments:
+        - `expr`: linear expression (sum, wsum, var)
+        
+        Returns tuple of:
+        - `expr`: linear expression (sum, wsum, var) without NegBoolView
+        - `const`: constant to be added to the the expression (or removed from the rhs of a comparison, as fit)
```

Comment:
This is what you should do with it, not __what__ it is exactly.
What I think it means: `cpm_expr == `expr + const`?

I.e., the difference between the original input expression and the transformed expression, resulting from rewring the expression using only positive Boolean variables